code/modules/mapping/reader.dm

```diff
@@ -401,6 +401,8 @@ var/global/dmm_suite/preloader/_preloader = new
 		var/value = attributes[attribute]
 		if(islist(value))
 			value = deepCopyList(value)
+		if(istext(value))
+			value = eval("\"[value]\"")
```

Comment:
undocumented != deprecated, eval as a feature is dead, it only exists because of byond's immense clinging to backwards compatibility.